styles/main.css

```diff
@@ -272,8 +377,13 @@ form p {
   padding-bottom: 50px;
 }
 
+.dynamics {
+  display: grid;
+}
+
 .card {
   display: flex;
+  justify-self: center;
```

Comment:
- [ ] [OPTIONAL] Our cards display now is somehow changed in desktop view and acting like the mobile version, Kindly remainder to fix it, It's content and image are not side by side as it was before in your latest commits and currently as you can see in the photo below, project image takes up big space I couldn't fit whole card in a single screenshot

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/156900361-a6520c57-1697-4761-b64c-5eacd75010b9.png) | ![image](https://user-images.githubusercontent.com/87689547/156900403-9c164c63-7dba-48be-b0f1-c33ee54045c4.png)

Reply:
I'm so sorry for not fixing this issue right now. But I still need to finish the other 3 project submissions to the reviewers. I'll fix it, as soon as possible.